gettsim/social_insurance/arbeitsl_v.py

```diff
@@ -112,12 +112,12 @@ def ges_beitr_arbeitsl_v_midi_jobreturn(
     -------
 
     """
-    bruttolohn_m_regulär_beschäftigt = bruttolohn_m.loc[regulär_beschäftigt]
-    bemess_grenze = rentenv_beitr_bemess_grenze.loc[regulär_beschäftigt]
-    return bruttolohn_m_regulär_beschäftigt.clip(upper=bemess_grenze)
+    bruttolohn_m_regulär_beschäft = bruttolohn_m.loc[regulär_beschäft]
```

Comment:
In my view, `reg_beschäftigt` is better than `regular_beschäft`

Reply:
I changed it.